tests/dir_helpers.py

```diff
@@ -66,9 +66,17 @@
 ]
 
 
-# see https://github.com/iterative/dvc/issues/3167
-git_logger = logging.getLogger("git")
-git_logger.setLevel(logging.CRITICAL)
+disabled_loggers = [
+    # see https://github.com/iterative/dvc/issues/3167
+    "git",
+    # see https://github.com/iterative/dvc/pull/3345/files#r380276443
+    # https://github.com/spulec/moto/issues/2535
+    "moto",
+]
+
+for logger_name in disabled_loggers:
```

Comment:
Btw, could we call `disable_other_loggers` here too? We are not using any 3rd party loggers in our tests anyway, so seems like we could disable them here as well.